lib/types/index.js

```diff
@@ -595,11 +595,12 @@ export type PatternStop = {
   id: string | number,
   pickupType: number,
   shapeDistTraveled: ?number,
+  stopHeadsign: ?string,
   stopId: string,
   stopSequence: number,
   stop_lat?: number,
   stop_lon?: number,
-  timepoint: ?number
+  timepoint: ?number,
```

Comment:
Revert this change

Reply:
Reverted in https://github.com/ibi-group/datatools-ui/pull/890/commits/ddf848719158b3500fd6520202ec781d5302dd5d